org-formation/050-costs/_tasks.yaml

```diff
@@ -0,0 +1,13 @@
+Parameters:
+  <<: !Include '../_parameters.yaml'
+
+AnomalyDetectorService:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.3.3/templates/Cost/anomaly-detector-service.yaml
+  StackName: !Sub '${resourcePrefix}-anomaly-detector-service'
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+    Account: '*'
+    Region: !Ref primaryRegion
+  Parameters:
```

Comment:
Do you know if the daily reports (the [current default value](https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/Cost/anomaly-detector-service.yaml#L9-L16) for the `Frequency` parameter) are sent regardless if there was a triggering event? I just don't want the root emails being spammed with empty reports.

Reply:
I  believe notifications will only trigger when the `threshold` is exceeded.. 
 https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ce-anomalysubscription.html#cfn-ce-anomalysubscription-threshold

Reply:
Got it!